src/edu/stanford/nlp/sempre/InteractiveMaster.java

```diff
@@ -0,0 +1,328 @@
+package edu.stanford.nlp.sempre;
```

Comment:
requires a lot of fiends in Parser, Response etc. to be changed to public... less sure about this one. Let me know which one you prefer.